_docs/advanced-features.md

```diff
@@ -163,10 +163,10 @@ For remote launches, we download the zip directory as returned by the Dockstore
 
 ## WDL Launcher Configuration
 
-By default, WDL tools/workflows will automatically be ran with [cromwell](https://github.com/broadinstitute/cromwell) 30.2.
+By default, WDL tools/workflows will automatically be ran with [cromwell](https://github.com/broadinstitute/cromwell) 36.
```

Comment:
Shouldn't this say "By default, WDL tools/workflows will automatically be **run** with [cromwell](https://github.com/broadinstitute/cromwell) 36."